express-hello-world/server.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
```

Comment:
Put a blank line after this. We'll be adding a lot more configuration and having a blank line will help make it clear where configuration stops and endpoints start.